### PR TITLE
chore: add Uffizzi demo button in README

### PR DESCRIPTION
**Pull request by gadkins**
## Summary

This PR adds a button to the project `README` which allows visitors to the logto project to get a live, interactive demo of logto with one click. Each demo instance is a sandboxed [Uffizzi](https://www.uffizzi.com) environment that is isolated from other demos, like all Uffizzi previews. This button requires **no configuration** by visitors, and it **does not** require them to have a Uffizzi account.  

[Demo](https://app.uffizzi.com/demo/github.com/logto-io/logto)

### How it works
This button makes a request to a public route `https://app.uffizzi.com/demo/github.com/logto-io/logto` which will trigger a demo deployment of logto on Uffizzi Cloud based on [this configuration](https://github.com/gadkins/logto/blob/uffizzi-live-demo-button/docker-compose.demo.uffizzi.yml). Note that if you want to demo a different tag than `logto-io/logto:prerelease`, let me know as I will need to make a change on the backend. You'll be able to make changes yourself in `docker-compose.demo.uffizzi.yml` in the next iteration.

## Testing

To test this PR you can click on the button above, or try it from my fork [here](https://github.com/gadkins/logto/tree/uffizzi-live-demo-button#interactive-demo).

### CODE REVIEW

**Comment by gao-sun** [comment]
@charIeszhao would you like to take a look? thanks

File: docker-compose.demo.uffizzi.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# This compose file is for demonstration only, do not use in prod.
+version: "3.9"
+
+x-uffizzi:
+  ingress:
+    service: app
+    port: 3001
+  continuous_previews:
+    deploy_preview_when_pull_request_is_opened: true
+    delete_preview_when_pull_request_is_closed: true
```
Comment (charIeszhao) [inline docker-compose.demo.uffizzi.yml:10]: Do we need to re-deploy each time a PR is opened or closed? Seems unnecessary to me as we are using `prerelease` build for the live demo

File: docker-compose.demo.uffizzi.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    share_to_github: true
```
Comment (charIeszhao) [inline docker-compose.demo.uffizzi.yml:11]: What does this config do?

File: docker-compose.demo.uffizzi.yml (hunk 1 above)
Comment (gadkins) [reply docker-compose.demo.uffizzi.yml:10]: These are not used. Will remove.

File: docker-compose.demo.uffizzi.yml (hunk 2 above)
Comment (gadkins) [reply docker-compose.demo.uffizzi.yml:11]: This is not used. Will remove. (It's for pull request commenting)

**Comment by gadkins** [comment]
This demo configuration setting is now available to members of the Logto team in the Uffizzi Dashboard here: https://app.uffizzi.com/projects/4064/settings/demo-button

**Review by charIeszhao — APPROVED** [review]

**Comment by charIeszhao** [comment]
Thank you, @gadkins! Much appreciated!

**Pull request merged**